src/geocat/viz/taylor/taylor.py

```diff
@@ -0,0 +1,600 @@
+        ----------
+        maintitle : str
+            Title text
+
+        fontsize : float, optional
+            Text fontsize
+
+        y_loc : float, optional, default to None
+            Vertical Axes location. 1.0 is the top.
+
+        *kwargs* are directly propagated to the `matplotlib.axes.Axes.set_title` command.
+
+        Return
+        ------
+        None
+        """
+
+        self._ax.set_title(maintitle, fontsize=fontsize, y=y_loc, **kwargs)
+
+    def set_fontsizes_and_pad(self,
+                              ticklabel_fontsize=14,
+                              axislabel_fontsize=16,
+                              axislabel_pad=8):
+        """Reset ticklabel and axis label fontsizes, and axis label padding.
+
+        Parameters
+        ----------
+        ticklabel_fontsize : float, optional, default to 14
+            Fontsize of all tick labels
+
+        axislabel_fontsize : float, optional, default to 16
+            Fontsize of axis labels
+
+        axislabel_pad : float, optional, default to 8
+            Padding between axis labels and axis
+
+        Return
+        ------
+        None
+        """
+
+        self._ax.axis['top', 'right',
+                      'left'].major_ticklabels.set_fontsize(ticklabel_fontsize)
+        self._ax.axis['top', 'right'].label.set_fontsize(axislabel_fontsize)
+        self._ax.axis['top', 'right'].label.set_pad(axislabel_pad)
+        
+###############################################################################
+
+def taylor_8():
+    # https://www.ncl.ucar.edu/Applications/Scripts/taylor_8.ncl
+    
+    # Case A                       
+    CA_std = [1.230, 0.988, 1.092, 1.172, 1.064, 0.990]
+    CA_corr = [0.958, 0.973, -0.740, 0.743, 0.922, 0.950]
+    #BA = [2.7, -1.5, 17.31, -20.11, 12.5 , 8.341]
+
+    # Case B
+    CB_std = [1.129, 0.996, 1.016, 1.134, 1.023, 0.962]
+    CB_corr = [0.963, 0.975, 0.801, 0.814, -0.946, 0.984]
+    
+    # Create a list of model names
+    namearr = ["Globe", "20S-20N", "Land", "Ocean", "N. America", "Africa"]
+    
+    # Create figure and TaylorDiagram instance
+    dia = TaylorDiagram()
+    
+    # Add model sets
+    modelTextsA, _ = dia.add_model_set(
+        CA_std, CA_corr,
+        13, (-3.5, 8), # mode label fontsize and offset from marker
+        model_outside_on=True,
+        marker='^',
+        s=60, # marker size
+        edgecolors='red',
+        facecolors='none',
+        label='Data A')
+    modelTextsB, _ = dia.add_model_set(
+        CB_std, CB_corr,
+        13, (-3.5, 8), 
+        model_outside_on=True,
+        marker='D',
+        s=60,
+        edgecolors='blue',
+        facecolors='none',
+        label='Data B')
+    
+    # Change properties of model labels to add bounding boxes
+    for txt in modelTextsA:
+        txt.set_bbox(dict(facecolor='red', edgecolor='none',
+                          pad=0.05, boxstyle='square'))
+        txt.set_color('white')
+    for txt in modelTextsB:
```

Comment:
I'm not sure if there is a way to edit the edge, face, and text color for case B data, but I do notice a difference between point 5 on NCL and this version. If there isn't a way to work around the colors being set for every point on the diagram, don't worry about it

Reply:
We could potentially, but I personally don't understand why the text format for the labels in the NCL is different. It seemed unnecessary to me.